android/PlayKit/app/src/main/AndroidManifest.xml

```diff
@@ -21,7 +21,7 @@
 
         <meta-data
             android:name="io.streamroot.dna.StreamrootKey"
-            android:value="YOUR_STREAMROOT_KEY" />
+            android:value="jeremybrixhe" />
```

Comment:
a forget :)

Reply:
You should rebase this off the commit list though.